### DISCUSSION TITLE
How is everything sorted?

### DISCUSSION THREAD

**Discussion by robman501a** [86Box/86Box#3137] [Q&A] [answered]
Is everything from machine type to other peripherals sorted from oldest to newest?

Example, for a 486 with no PCI or AGP slots:
Video: 
[ISA] ATI EGA Wonder 800+ (oldest?)
[VLB] Trident TGUI 9400AGi (newest?)

**Comment by TheVirtualComputer** [comment]
As far as I know, items are sorted alphabetically within groups with exception to those that are in the "Dev Branch".

**Reply by robman501a** [reply]
What's different about the "Dev Branch"?
How would I date the parts?

**Comment by TheVirtualComputer** [comment, marked as answer]
The "Dev Branch" has some features that are considered incomplete, such as the Matrox display adapters. Also, as for how you would date the various components, you can try to search for them online in order to find out when exactly they were released.
